experiments/customui/parent.js

```diff
@@ -292,11 +317,11 @@ var ex_customui = class extends ExtensionCommon.ExtensionAPI {
         
         sidebar = document.createXULElement("vbox");
         sidebar.setAttribute("persist", "width");
-        sidebar.setAttribute("width", "244");
```

Comment:
Did you test what happens if add-ons request unrealistic sizes, or multiple add-ons request different sizes? Bounding the size to some limit might be necessary to ensure the sidebar can always get closed by the user (I admit that forcing a specific width was the lazy way out though ;) ).

Reply:
I did now. (but not with multiple add-ons yet)
It's interesting to see. If the sidebar is too wide, it is indeed a problem to resize it to something reasonable.
I added a TODO in the code as I am not enough familiar with TB internals to implement that.